code-orange/DrawerView.swift

```diff
@@ -14,24 +14,21 @@ class DrawerView: UIView {
   public var contentView: UIView? {
     set {
       self.contentView?.removeFromSuperview()
-      self.hidingConstraint?.isActive = false
       guard let contentView = newValue else { return }
       install(contentView)
     }
     get {
-      return subviews.first
+      return containerView.subviews.last
     }
   }
   
-  public func show() {
-    alpha = 1 // TODO: use hiding constraint
-  }
-  
-  public func hide() {
-    alpha = 0 // TODO: use hiding constraint
-  }
-  
-  private var hidingConstraint: NSLayoutConstraint?
+  private lazy var containerView: UIView = {
+    let view = UIView()
+    view.layer.masksToBounds = true
```

Comment:
we use a view with corner radius nested inside a transparent view with shadow. We do it because we don't have a choice

https://stackoverflow.com/questions/3690972/why-maskstobounds-yes-prevents-calayer-shadow